Automattic/wp-calypso
# Concierge: Migrate Support Image to Webpack

#### Changes proposed in this Pull Request

This PR does a few things:

- Removes one of the support images, because a duplicate already exists (and the other is unused)
- Migrates them to Webpack; for details on the reasoning behind this, see #36850 
- Makes some alt text translatable (I think it's supposed to be translatable?) 

#### Testing instructions

There should only be three uses of this image: https://github.com/Automattic/wp-calypso/search?q=%22support.svg%22&unscoped_q=%22support.svg%22

<img alt="Screen Shot 2021-03-03 at 13 28 08" width="801" src="https://user-images.githubusercontent.com/17325/109734102-4f3d8c80-7c25-11eb-83ad-b9945f3e7d72.png">

All of them are in regards to Concierge, but admittedly, I'm unable to get them to appear. I tried playing around a bit with this file to no success: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/upsell-nudge/index.jsx

It would be great if the usages could be checked, but I'm pretty sure the code should be fine. The images should be completely identical. :)
